### ISSUE TITLE
Glasskube Support

### ISSUE THREAD

**Issue by secustor** [renovatebot/renovate#29322]
### Discussed in https://github.com/renovatebot/renovate/discussions/29206

<div type='discussions-op-text'>

<sup>Originally posted by **kosmoz** May 22, 2024</sup>
### Tell us more.

[Glasskube](https://github.com/glasskube/glasskube/) is a package manager for Kubernetes clusters. Glasskube and Glasskube packages can be managed via GitOps solutions, such as Flux or ArgoCD, in which case our users would love to get notified about updates directly in their repositories.</div>

**Comment by kosmoz** [comment]
Hi, we (Glasskube) would love to contribute this feature! 

Do you have any suggestions about where to best get started?

**Comment by rarkins** [comment]
Hi @kosmoz, that's great to hear! We recommend you implement in this order:
- Datasource
- Versioning
- Manager

Datasource or Datasource+versioning won't be particularly useful on their own, although potentially someone could already use our custom manager with them for some value. However it simplifies things to work on them and merge incrementally. The final validation would be a real e2e repo test once the manager PR is ready.

For datasource, check out [the readme](https://github.com/renovatebot/renovate/blob/main/lib/modules/datasource/readme.md) and then it can be easiest if you pick an existing datasource which is similar and start by copy/pasting. Here was the most recent datasource addition for reference: https://github.com/renovatebot/renovate/pull/27971

**Comment by kosmoz** [comment]
I already started working on the versioning scheme in parallel to the datasource, but I'm not sure where to push my changes to best fit your workflow. Should I create a new PR based on `main`?

**Comment by rarkins** [comment]
Yea there should be no overlaps
